docs/components/aws/elasticache.md

```diff
@@ -0,0 +1,792 @@
+&nbsp;
+
+#### Cluster Settings
+
+<p align="center">
+  <img style="Padding:20px" width="400" src="./images/elasticache/ecache_cluster_settings.png">
+</p>
+  
+
+Field               | Required           | Description
+---                 | :---:              | ---
+Cluster Name        | :white_check_mark: | Name for the cluster to be used as an identifier 
+Cluster Description | :white_check_mark: | Description for the cluster
+Port                | :white_check_mark: | The port number on which cluster accepts connections
+&nbsp;
+
+Field        | Constraints
+---          | ---
+Cluster Name | Name must contain from 1 to 20 alphanumeric characters or hyphens, the first character must be a letter, the name cannot end with a hyphen or contain two consecutive hyphens.
+&nbsp;
+
+#### VPC Configuration
+
+When creating a new ElastiCache cluster, you need to configure a VPC as well. Sigma enables
+functionality to create new VPC as well as use an existing one.
+
+##### Create New VPC
+
+To create new VPC, expand VPC settings and select **Create New VPC** option from VPC type dropdown.
+
+<p align="center">
+  <img style="Padding:20px" width="400" src="./images/elasticache/ecache_new_vpc.png">
+</p>
+
+Field                 | Required           | Description
+---                   | :---:              | ---
+VPC Name              | :white_check_mark: | Name of the VPC 
+CIDR block            | :white_check_mark: | IPv4 CIDR block for the VPC
+Subnet Name           | :white_check_mark: | Name of the private subnet 
+CIDR block for subnet | :white_check_mark: | IPv4 CIDR block for the private subnet
+
+**Important:**  
+> **When creating a new VPC, make sure your lambda is also deploying in the same VPC. Select No VPC option
+from lambda Advanced configuration. Then Sigma IDE will be automatically deployed your lambda in the new VPC you created.**
+
+##### Using Existing VPC
+
+To use already existing VPC, select **Existing VPC** option from VPC type under VPC settings.
+
+<p align="center">
+  <img style="Padding:20px" width="400" src="./images/elasticache/ecache_existing_vpc.png">
+</p>
+
+Field          | Required           | Description
+---            | :---:              | ---
+VPC            | :white_check_mark: | Select a VPC from available VPC dropdown 
+Subnet         | :white_check_mark: | Select a subnet from available subnets for selected VPC 
+Security Group | :white_check_mark: | Select a security group from available security groups for selected VPC 
+
+**Important:**  
+> **When using an existing VPC, make sure your lambda function is also deployed in the same VPC. Either select the same VPC 
+(with the same Subnet and Security Group) that you selected for ElastiCache cluster in lambda advanced configuration or 
+leave 'No Vpc' option so that Sigma IDE will be able to deployed your lambda function in the same VPC as ElastiCache cluster**
```

Comment:
_L_ambda (check all)

for _the_ Elasti..

_A_dv.. _C_onf..

maybe don't make the whole sentence boldface (since "Important" is already in bold); instead make just boldface the important phrases, e.g.
**No VPC** (bold, case)

able to deploy (delete "ed")

as _the_ Elasti.. cluster_._